gamestonk_terminal/stocks/behavioural_analysis/ba_api.py

```diff
@@ -66,3 +66,5 @@
 from gamestonk_terminal.stocks.behavioural_analysis.finnhub_view import (
     display_stock_price_headlines_sentiment as snews,
 )
+
+from .cramer_view import display_cramer_daily as cramer
```

Comment:
Can we stick with full path here? for consistency

Reply:
lol.  just copied it over

<img width="598" alt="Screen Shot 2022-03-10 at 9 50 10 AM" src="https://user-images.githubusercontent.com/18151143/157687163-a5f19087-c1da-4b01-b182-501212795222.png">